packages/node/src/workers/cloud-platforms/aws.test.ts

```diff
@@ -1,3 +1,5 @@
+import { BigNumber } from 'bignumber.js';
```

Comment:
I think we should be explicit about this dependency in the `node` package's package.json. Can we add `bignumber.js` there too please?